src/UI/api/queries.ts

```diff
@@ -53,6 +35,26 @@ query Occurrence {
 }`;
 };
 
+export const occurrenceFilterQuery = (
```

Comment:
Is this the right name for this query? Looks like it returns the CSV data - I'd expect that to be reflected in the name somehow.